Dumbledraw/style.yaml

```diff
@@ -0,0 +1,106 @@
+xlabeling:
+  mt:
+    m_vis:
+      label: Visible di-#tau mass / GeV
+    pt_1:
+      label: Muon p_{T} / GeV
+    pt_2:
+      label: Tau p_{T} / GeV
+    jpt_1:
+      label: Leading Jet p_{T} / GeV
+    jpt_2:
+      label: Trailing Jet p_{T} / GeV
+    jphi_1:
+      label: Leading Jet #phi
+    jphi_2:
+      label: Trailing Jet #phi
+    njets:
+      label: Number of Jets
+    nbtag:
+      label: Number of b-tagged Jets
+    mt_1:
+      label: Transverse Mass m_{T} (#mu, #slash{E}_{T}) / GeV
+    mt_2:
+      label: Transverse Mass m_{T} (#tau, #slash{E}_{T}) / GeV
+    ptvis:
+      label: Visable p_{T} / GeV
+    pt_tt:
+      label: p_{T}(#tau#tau) / GeV
+    mjj:
+      label: Di-jet Mass m_{jj} / GeV
+    dijetpt:
+      label: Di-jet p_{T} / GeV
+    met:
+      label: Missing transversal Energy / GeV 
+  et:
+    m_vis:
+      label: Visible di-#tau mass / GeV
+    pt_1:
+      label: Electron p_{T} / GeV
+    pt_2:
+      label: Tau p_{T} / GeV
+    jpt_1:
+      label: Leading Jet p_{T} / GeV
+    jpt_2:
+      label: Trailing Jet p_{T} / GeV
+    jphi_1:
+      label: Leading Jet #phi
+    jphi_2:
+      label: Trailing Jet #phi
+    njets:
+      label: Number of Jets
+    nbtag:
+      label: Number of b-tagged Jets
+    mt_1:
+      label: Transverse Mass m_{T} (#mu, #slash{E}_{T}) / GeV
+    mt_2:
+      label: Transverse Mass m_{T} (#tau, #slash{E}_{T}) / GeV
+    ptvis:
+      label: Visable p_{T} / GeV
+    pt_tt:
+      label: p_{T}(#tau#tau) / GeV
+    mjj:
+      label: Di-jet Mass m_{jj} / GeV
+    dijetpt:
+      label: Di-jet p_{T} / GeV
+    met:
+      label: Missing transversal Energy / GeV 
+
+legend_label:
+    HTT: H#rightarrow#tau#tau
+    ggH: ggH
+    qqH: qqH
+    ZTT: Z#rightarrow#tau#tau
+    ZLL: Z#rightarrowll
+    ZL: Z#rightarrowll (l#rightarrow#tau_{h})
+    ZJ: Z#rightarrowll (jet#rightarrow#tau_{h})
+    TT: t#bar{t}
+    TTT: t#bar{t} (t#rightarrow#tau_{h})
+    TTJ: t#bar{t} (jet#rightarrow#tau_{h})
+    W: W+jets
+    WT: (W#rightarrow#tau)+jets
+    WL: (W#slash{#rightarrow}#tau)+jets
+    VV: Di-boson
+    QCD: QCD multijet
+    EWK: Electroweak
+    EWKZ: EWKZ
+    data_obs: Data
+
+color_dict:
```

Comment:
Suggestion: keep color_dict in styles.py in order to avoid the eval command and call this file labels.yaml, also in order to avoid the ambiguity between styles.py and styles.yaml